tests/crowdsourcing/tasks/model_chat/expected_states/state.json

```diff
@@ -272,6 +272,7 @@
                         "annotations_config": "{\n    \"config\": {\n        \"bucket_0\": {\n            \"name\": \"Bucket 0\",\n            \"description\": \"this response implies something...0\"\n        },\n        \"bucket_1\": {\n            \"name\": \"Bucket 1\",\n            \"description\": \"this response implies something...1\"\n        },\n        \"bucket_2\": {\n            \"name\": \"Bucket 2\",\n            \"description\": \"this response implies something...2\"\n        },\n        \"bucket_3\": {\n            \"name\": \"Bucket 3\",\n            \"description\": \"this response implies something...3\"\n        },\n        \"bucket_4\": {\n            \"name\": \"Bucket 4\",\n            \"description\": \"this response implies something...4\"\n        },\n        \"none_all_good\": {\n            \"name\": \"None, all good\",\n            \"description\": \"This response implies that there are no problems with the data\"\n        }\n    },\n    \"show_line_breaks\": false\n}\n",
                         "model_nickname": "fixed_response",
                         "model_opt": {
+                            "is_debug": false,
```

Comment:
Hmm - this file and state__image_chat.json can also be deleted because they've been supplanted by the outputs from pytest regressions, right?